programs/redstone-sol/src/util.rs

```diff
@@ -0,0 +1,66 @@
+pub type U256 = [u8; 32];
+
+pub trait Trim<T>
+where
+    Self: Sized,
+{
+    fn trim_end(&mut self, len: usize) -> T;
+}
+
+impl Trim<Vec<u8>> for Vec<u8> {
+    fn trim_end(&mut self, len: usize) -> Self {
+        if len >= self.len() {
+            std::mem::take(self)
```

Comment:
is using `std` recommended for solana? For example for Casper there is "their own std"?

Reply:
std works fine, we could use `core::mem::take` but both options work, no difference

I am using a HashSet for verifying signer count, also from `std`